dev/bots/docs.sh

```diff
@@ -14,8 +14,27 @@ FLUTTER_ROOT=$PWD dart dev/tools/javadoc.dart
 cp dev/docs/google2ed1af765c529f57.html dev/docs/doc
 
 # Upload new API docs when on Travis and branch is master
-
 if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then
   cd dev/docs
   firebase deploy --project docs-flutter-io
+else
```

Comment:
I don't think `else` will work here, as there are many configs travis can run in. For example, we don't want to run this on API jobs.

I think you need `[ "$TRAVIS_PULL_REQUEST" = "true" ] && [ "$TRAVIS_BRANCH" = "master" ];`